### PR TITLE
remove async await void

### PR DESCRIPTION
**Pull request by ayocodingit**
## Overview

- remove async await void

cc: @jabardigitalservice/jds-backend 

## Evidence
- title: remove async await void
- project: Digiteam
- participants: @firmanalamsyah580 @sandisunandar99 @novansyaah @khihadysucahyo @budiaramdhanrindi @f1reFr0st

### CODE REVIEW

**Review by setiadijoe — APPROVED** [review]

**Review by rachadiannovansyah — APPROVED** [review]
it's credit thanks to @khihadysucahyo who's reminding that

**Comment by ayocodingit** [comment]
> it's credit thanks to @khihadysucahyo who's reminding that

thanks to @khihadysucahyo

**Review by sandisunandar99 — APPROVED** [review]

**Review by rindibudiaramdhan — APPROVED** [review]

**Comment by rindibudiaramdhan** [comment]
May I know why await is removed @ayocodingit ?

**Comment by ayocodingit** [comment]
> May I know why await is removed @ayocodingit ?

because shouldn't need to use async await for void method

File: utils/verifySecretKey.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import config from '../config'
 import lang from './lang'
 
-const verifySecretKey = async (secret: string): Promise<void> => {
+const verifySecretKey = (secret: string): void => {
   if (secret !== config.get('secret.key')) throw Error(lang.__('credential_failed'))
 }
```
Comment (khihadysucahyo) [inline utils/verifySecretKey.ts:6]: just for discuss, I think checker function like this better if we considering to return an error even just using only bool, or (error, bool). wondering if this function can be more reuseable for any usecase.

so maybe u can call this function like this
```
if isVerfiedSecretKey {
 ....
}
```

**Comment by khihadysucahyo** [comment]
anyway, nice improvement dude

**Review by khihadysucahyo — APPROVED** [review]

**Pull request merged**